src/app/common/directives/feature-flag.directive.ts

```diff
@@ -15,6 +15,7 @@ export class FeatureFlagDirective {
 
     @Input("appFeatureFlag") public featureName: string;
 
+    /* istanbul ignore next */
```

Comment:
Why?

Reply:
I thought that this affects the percentage of test coverage, so I deleted it.